tests/server/3-document.spec.js

```diff
@@ -198,7 +198,6 @@ describe('Document', () => {
           .get('/api/documents?role=regular')
           .set('authorization', userToken)
           .end((err, res) => {
-              console.log(res.body);
               res.status.should.equal(200);
               for(let i = 0; i < res.body.length; i++) {
```

Comment:
### Critical

You conveniently ignored this feedback I gave in an earlier PR #12 . Look here for more clarity on the subject and rectify accordingly. Airbnb Rule - Keep it functional